### PR TITLE
Nodejs week3

### PR DESCRIPTION
**Pull request by andrii1**

### CODE REVIEW

File: nodejs/week3/readme.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# Readme
\ No newline at end of file
+# Readme
+
+Example of SQL injection in Knex
+https://www.loom.com/share/fcbd3b29ea824a27a72898cf1c877848
```
Comment (itsgauravsaxena) [inline nodejs/week3/readme.md:4]: Very well done!!

**Review by itsgauravsaxena — COMMENTED** [review]
I dont see how you fixed sql injection issue in app.js or did I miss something?

**Review by 27ManishaPatel — APPROVED** [review]
Awesome well done !

**Comment by andrii1** [comment]
> I dont see how you fixed sql injection issue in app.js or did I miss something?

I did this: multipleStatements: false,
But, apart from this I just used official knex statements hoping it will prevent sql injections :) What should be the best way?

**Pull request merged**